Source/WebCore/platform/graphics/FontPlatformData.cpp

```diff
@@ -114,4 +115,68 @@ Vector<FontPlatformData::FontVariationAxis> FontPlatformData::variationAxes(Shou
 }
 #endif
 
+#if USE(CORE_TEXT)
+bool FontPlatformData::canMakeBaseFontDescriptor(const Vector<uint8_t>& fontFaceData, String& itemInCollection)
+{
+    Vector<uint8_t> copyOfFontFaceData = fontFaceData;
+    auto localFontFaceData = SharedBuffer::create(WTFMove(copyOfFontFaceData));
+    auto fontCustomPlatformData = createFontCustomPlatformData(localFontFaceData, itemInCollection);
+    if (!fontCustomPlatformData)
+        return false;
+    auto baseFontDescriptor = fontCustomPlatformData->fontDescriptor.get();
+    if (!baseFontDescriptor)
+        return false;
+    return true;
+}
```

Comment:
Seems kinda wasteful to go through the creation of a `FontCustomPlatformData` and not use it. Not sure what the better way is here just making a note.